index.html

```diff
@@ -666,6 +678,45 @@ <h2 id="AudioContext">
           cases, only a single <a><code>AudioContext</code></a> is used per
           document.
         </p>
+        <p>
+          <!-- ReSpec does not support documenting constructor -->
+           <span class="synchronous">When creating an <a>AudioContext</a>,
+          execute these steps:</span>
+        </p>
+        <ol>
+          <li>Set a <code>control thread flag</code> to
```

Comment:
Elsewhere you refer to this as a "control thread state" flag, should be consistent.